setup.py

```diff
@@ -64,6 +64,6 @@ def get_install_requires():
         'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
         'Operating System :: OS Independent',
         'Framework :: Django',
-        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3',
```

Comment:
Can you update this consistently with the other modules please?

Reply:
We don't mention the Python version in any OpenWISP module. I checked the following 

- https://github.com/openwisp/openwisp-utils/blob/master/setup.py
- https://github.com/openwisp/openwisp-users/blob/master/setup.py
- https://github.com/openwisp/openwisp-controller/blob/master/setup.py
- https://github.com/openwisp/openwisp-ipam/blob/master/setup.py

Reply:
Ok great, maybe the other modules I saw were libraries and we use more classifiers there.